runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_crafting.json

```
@@ -66,12 +78,36 @@
       "name": "Leather Boots",
       "xp": 16.3
     },
+    {
+      "level": 7,
+      "icon": 1789,
+      "name": "Unfired pie dish",
+      "xp": 10
```

This should be 15: https://oldschool.runescape.wiki/w/Unfired_pie_dish
```suggestion
      "xp": 15
```